### PR TITLE
0.13.2-0.0.3: [enhancement] - Add popular networks support, reformatting using prettier

### PR DESCRIPTION
**Pull request by Adamj1232**
Add popular networks support
Reformatting using prettier

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -74,6 +74,7 @@ const App = () => {
           )
 
           window.localStorage.setItem('selectedWallet', wallet.name)
+          console.log(wallet)
```
Comment (aaronbarnardsound) [inline src/App.js:77]: Need to remove this log :)

File: src/App.js [hunk 2]
```diff
@@ -171,13 +172,15 @@ const App = () => {
       value: 1000000000000000
     }
 
-    const sendTransaction = () =>
+    const sendTransaction = () => {
```
Comment (aaronbarnardsound) [inline src/App.js:175]: This function needs to return the result from the call to `signer.sendTransaction`

File: src/App.js [hunk 3]
```diff
@@ -171,13 +172,15 @@ const App = () => {
...
       signer.sendTransaction(txDetails).then(tx => tx.hash)
+    }
 
     const gasPrice = () => provider.getGasPrice().then(res => res.toString())
 
-    const estimateGas = () =>
+    const estimateGas = () => {
```
Comment (aaronbarnardsound) [inline src/App.js:181]: This also needs to return the result of the call to `estimateGas`

**Review by aaronbarnardsound — CHANGES_REQUESTED** [review]
Just a couple of small changes needed.

File: src/App.js (hunk 2 above)
Comment (Adamj1232) [reply src/App.js:175]: ahh yeah whoops! Good catch!! Ill make sure my formatter on this machine is doin wonky stuff like this in the future.

**Review by aaronbarnardsound — APPROVED** [review]
Looks good thanks!

**Pull request merged**